release/nightly_tests/stress_tests/placement_group_tests_compute.yaml

```diff
@@ -0,0 +1,28 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
+region: us-west-2
+
+max_workers: 5
+
+aws:
+    BlockDeviceMappings:
+        - DeviceName: /dev/sda1
+          Ebs:
+            VolumeSize: 500
+
+head_node_type:
+    name: head_node
+    instance_type: m4.16xlarge
+    resources:
+      cpu: 64
+
+worker_node_types:
```

Comment:
@rkooo567  we created 5 worker nodes as the test_replacement group is attaching to prod cluster instead of local started simulating clusters; let me know if this makes sense.

Reply:
Oh I see. Yeah this is actually better sgtm